src/cmdc_tools/datasets/official/base.py

```diff
@@ -10,6 +10,7 @@
 class CountyData(InsertWithTempTable, ABC):
     table_name: str = "us_covid"
     pk: str = '("vintage", "dt", "fips", "variable_id")'
+    provider = "state"
```

Comment:
I made these changes in the provider branch

I vote we revert them here so we can keep provider details in that PR

Reply:
Done.